### DISCUSSION TITLE
我想使用chatgpt web这个项目的前端,并加入可以在前端调换“创造”、“平衡”、“严谨”的小模块

### DISCUSSION THREAD

**Discussion by Hanxu4131** [acheong08/EdgeGPT#297] [Q&A]
我想使用下面这个项目的前端,有人知道如何做吗,或者有现成的方案吗,感谢.
https://github.com/gouguoyin/chatgpt-web

**Comment by Nu11Const** [comment]
有现成方案，不过目前并不完善。
这是我写的:[URL](https://github.com/VMTask/NewBing-Web)

**Reply by Nu11Const** [reply]
而且跟那个的界面也不太一样。
